hack/test/gke/xfs-formatter/tune2fs_test.sh

```diff
@@ -47,7 +47,7 @@ for t in "minix" "msdos"; do
 done
 
 echo "${message_prefix}..testing valid tune2fs filesystems." >&2
-for t in "ext2" "ext3" "ext4" "xfs"; do
```

Comment:
- Don't waste CI cycles
- Don't assume 'ext2' or 'ext3' are valid filesystems for our use cases.
- Won't be broken when ext2 dies (die! die already!)
etc.

Reply:
> Don't waste CI cycles

although this is limiting to 0

> Don't assume 'ext2' or 'ext3' are valid filesystems for our use cases.

To my knowledge, this is replacing the tune2fs binary used by GKE setup script that we don't control. It's not about what filesystems we like but about keeping their scripts working. A concern that makes me not want to touch this too much to avoid a big load of testing and possible issues in the future updates.

> Won't be broken when ext2 dies (die! die already!)

That could be valid, although I am not sure how soon it comes, but say ext3 has often been used for a boot partition even if the main disks got ext4.

I think in general our long term hope is to drop this when GKE will allow us to pick xfs as the filesystem.

Reply:
I can remove this change - really it doesn't matter that much. The point of the patchset was the fix the 'dd' command inefficiency, this was a minor nit.